cmd/prometheus/main.go

```diff
@@ -793,6 +798,8 @@ func main() {
 		cancel := make(chan struct{})
 		g.Add(
 			func() error {
+				readyStatus.Set(0)
```

Comment:
Added `set(0)` when we receive `SIGTERM`

Reply:
it then does not match the /-/ready endpoint

Reply:
ah, sorry, reverted then

Reply:
I think those should match. Probably it would make sense to change /-/ready when we quit as well, we could do it in a 2nd PR